modules/tinymce/src/core/main/ts/DragDropOverrides.ts

```diff
@@ -21,6 +21,11 @@ import * as Predicate from './util/Predicate';
  * @class tinymce.DragDropOverrides
  */
 
+// Arbitrary values needed when scrolling CEF elements
+const ScrollingValue = 32;
+const IntervalValue = 100;
+const MouseRange = 4;
```

Comment:
You refer to these as arbitrary, yet i assume they actually do affect the behavior. Do we want to include a proper description of what they do, or even make them options? And if not, how well does the behavior match CET elements being dragged, and how do we refine it? Should these depend on the browser used, to better match behavior there?

Reply:
Arbitrary doesn't mean random, it just means they could have been different but I believe these values to be sane defaults.
Yes we can include a proper description of what they affect. As for making them options configurable by the client, I don't think that would be necessary but that's more up to the product team to decide.

> how well does the behavior match CET elements being dragged, and how do we refine it? 

Well, Safari doesn't scroll when dragging CET elements so the behavior doesn't match there. For other editors it seems to me to match well but that can be debated. We refine the behavior through the use/change of these variables.

> Should these depend on the browser used, to better match behavior there?

Yes we can make them depend in the browser, but the behavior doesn't change much between chrome and firefox. So having a sane default can suffice, I think.